core/src/test/java/com/linecorp/armeria/common/stream/DefaultByteStreamMessageTest.java

```diff
@@ -95,7 +95,7 @@ void skip1_take1() {
         final ByteStreamMessage byteStreamMessage =
                 ByteStreamMessage.of(delegate).range(1, 1);
         StepVerifier.create(byteStreamMessage, 1)
-                    .expectNext(HttpData.ofUtf8("2"))
+                    .expectNextMatches(httpData -> "2".equals(httpData.toStringUtf8()))
```

Comment:
IIRC, we can revert these changes because we fixed the `equals` implementation to check the actual content.